interfaces/DefaultWidget.ts

```diff
@@ -120,6 +120,16 @@ const MAPPING_EN: PerLanguageMapping = new Map([
 		`Paris is the <mask> of France.`,
 		`The goal of life is <mask>.`,
 	] ],
+	[ "sentence-similarity", [
+		{
+			source_sentence: "That is a happy person",
+			sentences: [
+				"That is a happy dog",
+				"That is a very happy person",
+				"Today is a sunny day"
+			]
+		},
+	] ],
```

Comment:
can we do a couple more languages, e.g. if we already have sentence-similarity models trained in those languages?

Reply:
At the moment we don't have any non-English models configured in the Hub (we need to re-upload the models with `sentence-transformers` v2 release), but I'm happy to add some translations for other languages since they will be useful :)

I added Chinese, French, Spanish, Russian, Italian, Arabic, Bengali, Mongolian and German.